src/TimeSync.cpp

```diff
@@ -34,35 +39,51 @@ namespace time_sync {
                 std::string performance_logging = getSystemConfig("PERFORMANCE_LOGGING","FALSE");
                 _performance_logging = performance_logging.compare("true") == 0 || performance_logging.compare("TRUE") == 0 ;
                 if (_performance_logging) {
-                    SPDLOG_DEBUG("Performance logging is enabled!");
-                    auto logger = createLogger("performance", ".csv", "%v", spdlog::level::info);
-                    logger->info("Real Time (ms), Carma Time (ms)");
+                    logDebug("Performance logging is enabled!");
+                    if (!createPerformanceLogger()) {
+                        logDebug("Failed to create performance logger!");
+                    }
                 }
                 // Initalize kafka consumer
-                SPDLOG_INFO("TimeSync is in simulation mode!");
+                logDebug("TimeSync is in simulation mode!");
                 _time_consumer= std::make_unique<udp_socket::UdpServer>(_ip, _port);
                 consumer_thread = std::thread(&TimeSync::consumeTimeLoop, this);
                 consumer_thread.detach();
             }
         }
     }
+
+    void TimeSync::logDebug(const std::string &msg) const{
+        if (_debug) {
+            std::cout << "[DEBUG]"  <<  msg << std::endl;
```

Comment:
A minor comment: Common elements in a log format include a timestamp, log level, source file and line number, and the actual message.

Reply:
Yeah without a setup logger might be involved to get line number and file. Normally log statements with line number and file are meant for developers. Many debug log statements for end user use do not include this type of information. We will treat these log statements as such